app/components/Inspector/HighlighterRect.js

```diff
@@ -27,8 +27,10 @@ export default class HighlighterRect extends Component {
 
       // Add class + special classes to hovered and selected elements
       let highlighterClasses = [InspectorCSS['highlighter-box']];
-      hoveredPath === node.path && highlighterClasses.push(InspectorCSS['hovered-element-box']);
-      selectedPath === node.path && highlighterClasses.push(InspectorCSS['inspected-element-box']);
+      if (hoveredPath === node.path)
```

Comment:
It's just an appium style thing but we've opted to never do brace-less ifs, with one exception:

```js
if (err) return cb(err);
```

is ok. I know this is safe but to keep with consistency, mind if we add braces?